config/locales/en.yml

```diff
@@ -2721,6 +2721,101 @@ See the %{link} to find out more about %{sitename}'s features and to start using
       have_an_account: "Already have an account?"
       action_login: "Log in now."
 
+  # Most popular names used in variant_unit_name.
+  # We use these entries to pluralize unit names in every language.
+  #
+  # Extracted with the following query:
+  # Spree::Product.group(:variant_unit_name).order("count_all DESC").count.each { |name, count|
+  #   puts "    # Used #{count} times."
+  #   puts "    #{name&.parameterize('_')}:"
+  #   puts "      one: \"#{name}\""
+  #   puts "      other: \"#{name}s\"";
+  # }
+  unit_names:
+    # Used 379 times.
```

Comment:
I dont think we need to keep these numbers in this file. The numbers are useful to see what keys to have here but we I think we should remove these "statistics" for here.

Reply:
Okay, I thought it could be interesting to compare if we update it one day. But I'll remove them.